client/src/app/site/pages/meetings/pages/interaction/services/interaction.service.ts

```diff
@@ -179,4 +183,15 @@ export class InteractionService {
             });
         }
     }
+
+    private isFirstInteractionService(): boolean {
+        for (let i = 0; i < InteractionServiceRegistry.length; i++) {
+            if (InteractionServiceRegistry[i] === this) {
+                break;
+            } else if (!!InteractionServiceRegistry[i]) {
```

Comment:
When would this ever happen? `this` is the only thing added and that can never be `null`, right?